src/ui/components/util/Localization.ts

```diff
@@ -0,0 +1,77 @@
+import type { PlayerError, VideoQuality } from 'react-native-theoplayer';
+
+export type Localization = Partial<AllLocalization>;
+
+export interface AllLocalization {
```

Comment:
We should document all the properties here, so customers can understand where and how these translations are used. Have a look at [the Android UI](https://github.com/THEOplayer/android-ui/blob/v1.11.0/ui/src/main/res/values/strings.xml) for some examples.